include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -375,6 +380,7 @@ namespace internal
       using value_type               = Tensor<1, dim, Number>;
       using scalar_value_type        = Tensor<1, dim, ScalarNumber>;
       using vectorized_value_type    = Tensor<1, dim, VectorizedArrayType>;
+      using unit_gradient_type       = Tensor<2, dim, Number>;
```

Comment:
I would remove this here as this is a special case `dim = spacedim`.

Reply:
I need to check this more.